src/Toolkit/Toolkit.Maui/Compass/CompassHandler.cs

```diff
@@ -27,131 +27,132 @@
 using NativeViewType = System.Object;
 #endif
 
-namespace Esri.ArcGISRuntime.Toolkit.Maui.Handlers
+namespace Esri.ArcGISRuntime.Toolkit.Maui.Handlers;
+#pragma warning disable CS1591

```

Comment:
This missing comment is an oversight - we should fix that instead of hiding. The class needs to be documented.